deployments/create-k8s.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+rm -rf ./k8s
+mkdir -p ./k8s
+
+# Remove restart policy from all services
+# Set controller type to statefulset for apis, which maintain a cache on disk
+# Set service type to loadbalancer for web service, which should be exposed outside the cluster
+yq \
+    '(del(.services[].restart)) |
+     ((.services | (.billing-api, .main-api)).labels += {"kompose.controller.type":"statefulset"}) |
+     (.services.web.labels += {"kompose.service.type":"loadbalancer"})
+     ' \
+    docker-compose-split.yaml | \
+    kompose -f - -o k8s convert -n oms --with-kompose-annotation=false
+
+# Rename the web-tcp service to web
```

Comment:
Would be better for some of these comments to be "why" comments rather than/in addition to "what" comments.